homeassistant/components/alexa/smart_home_http.py

```diff
@@ -59,6 +60,14 @@ def locale(self):
         """Return config locale."""
         return self._config.get(CONF_LOCALE)
 
+    @property
+    def camera_stream_url(self):
```

Comment:
There is no need for this. Just hardcode it to `self.hass.config.api.base_url`. Since Alexa will always be local. https://github.com/home-assistant/core/blob/dev/homeassistant/components/google_assistant/trait.py#L251